packages/three-mmd/src/loaders/MMDLoader.ts

```diff
@@ -194,7 +217,7 @@ class CubicBezierInterpolation extends Interpolant {
     const eps = 1e-5
     const math = Math
 
-    let sst3, stt3, ttt
+    let sst3 =-1, stt3 = -1, ttt = -1
```

Comment:
why `-1`?

Reply:
-1 was to make their type `number`. the actual values always get overwritten.